listenbrainz/webserver/views/api_compat_deprecated.py

```diff
@@ -126,6 +126,8 @@ def submit_listens():
         return 'FAILED Invalid data submitted!\n', 400
 
     user = db_user.get(db_conn, session.user_id)
+    if user['is_paused']:
+        raise APIUnauthorized("user_id is paused and is currently not accepting listens. Feel free to contact us if you have any questions about this. https://metabrainz.org/contact")
```

Comment:
Like other errors in this api file, this should be a plain text message with the error code.